### PR TITLE
Development

### PR DESCRIPTION
**Pull request by osinakayah**

### CODE REVIEW

**Comment by talihomz** [comment]
Hi @osinakayah 

This runs ok, was able to create a user, as well as play with the associations for Posting and commenting, great work.

There are a few things we that need clarification in the code.

File: app/controllers/comments_controller.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+class CommentsController < ApplicationController
+
+
+  include CommentsHelper
+  def create
+    @comment = Comment.new(comment_params)
+    @comment.user_id = 1
```
Comment (talihomz) [inline app/controllers/comments_controller.rb:8]: Why is this `user_id` hard coded?

File: app/controllers/posts_controller.rb [hunk 2]
```diff
@@ -0,0 +1,41 @@
+class PostsController < ApplicationController
+  include PostsHelper
+
+  def index
+    @posts = Post.all
+  end
+
+  def new
+    @post = Post.new
+    @user = User.find(1)
```
Comment (talihomz) [inline app/controllers/posts_controller.rb:11]: Why is this `user_id` hard-coded?

File: app/controllers/posts_controller.rb [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+  end
+  def update
+    @post = Post.find(params[:id])
+    @post.update(post_params)
+    redirect_to post_path(@post)
+  end
+  def show
+    @post = Post.find(params[:id])
+    @comment = Comment.new
+    @comment.user_id = 1
```
Comment (talihomz) [inline app/controllers/posts_controller.rb:38]: Hard-coded `user_id`.

**Comment by sanelca** [comment]
@talihomz 
"There are a few things we that need clarification in the code."
Why you did not write your comments about code here?

**Comment by osinakayah** [comment]
@talihomz Please, what are the things you need clarifications for?

**Review by talihomz — CHANGES_REQUESTED** [review]

**Comment by talihomz** [comment]
Hi @osinakayah @osinakayah. I'm terribly sorry for leaving you hanging on this!

On Github, when you click on "Start Review", apparently one also needs to finalize by clicking "Request Changes" for them to appear here, and I was only aware of this now! 🙏🏼
 
I've posted them here, and again I am terribly sorry about this!

File: app/controllers/comments_controller.rb (hunk 1 above)
Comment (sanelca) [reply app/controllers/comments_controller.rb:8]: That is a really big issue...  Do you have any another comment like that? Thank you... We will resolve that.

File: app/controllers/posts_controller.rb (hunk 2 above)
Comment (sanelca) [reply app/controllers/posts_controller.rb:11]: We will replace that...

File: app/controllers/posts_controller.rb (hunk 3 above)
Comment (sanelca) [reply app/controllers/posts_controller.rb:38]: Thank you.

**Comment by sanelca** [comment]
> Hi @osinakayah @osinakayah. I'm terribly sorry for leaving you hanging on this!
> 
> On Github, when you click on "Start Review", apparently one also needs to finalize by clicking "Request Changes" for them to appear here, and I was only aware of this now! 🙏🏼
> 
> I've posted them here, and again I am terribly sorry about this!

It is Ok we will resolve issues and make new review request :)

**Pull request closed without merging**